Integrations/integration-CyberTriage.yml

```diff
@@ -180,25 +182,31 @@ script:
         is_true = lambda x: x == 'yes'
         is_hash_upload_on = is_true(CMD_ARGS['malware_hash_upload']) # arg value = 'yes' or 'no'
         is_file_upload_on = is_true(CMD_ARGS['malware_file_upload']) # arg value = 'yes' or 'no'
-        is_fast_scan_on = not(is_true(CMD_ARGS['full_scan']))        # arg value = 'yes' or 'no'
```

Comment:
Why did you remove this?

Reply:
Fast scan is no longer supported by Cyber Triage. It was replaced with scanOptions.

Reply:
I understand, but removing it will break backward compatibility.
Does `fs` in `scanOptions` replace it?